File: Problem/pool/PRIM0b/sol/c++/ac/4.cc
#include <cstdio>
bool isPrime(int x){
  if(x<2){return false;}//0,1
  for(int d=2;d<x;d++){
    if(0==x%d){
      return false;
    }
  }
  return true;
}

int main(){
  int x;
  while(1==scanf("%d",&x)){
//printf("%d: ",x);
    printf("%s\n",isPrime(x)?"prím":"nem prím");
  }

  return 0;
}

File: Front/problem/name/PRIM/sol/julia/ac/4.jl
function isPrime(x)
  ans=false
  if x>1 
    ans=true
    for d in 2:(x-1)
      if 0==mod(x,d)
        ans=false
        break
      end
    end
  end
  ans
end

for x in split(readline())
  if true==isPrime(parse(Int,x))
    println("prím")
  else
    println("nem prím")
  end
end
